src/languages/nl_NL.js

```diff
@@ -0,0 +1,112 @@
+export default {
+    languageName: "Nederlands",
+    checkEverySecond: "Controleer elke {0} seconden.",
+    "Avg.": "Gem. ",
+    retriesDescription: "Maximum aantal nieuwe pogingen voordat de service wordt gemarkeerd als niet beschikbaar en er een melding wordt verzonden",
+    ignoreTLSError: "Negeer TLS/SSL-fout voor HTTPS-websites",
+    upsideDownModeDescription: "Draai de status om. Als de service bereikbaar is, is deze OFFLINE.",
+    maxRedirectDescription: "Maximaal aantal te volgen omleidingen. Stel in op 0 om omleidingen uit te schakelen.",
+    acceptedStatusCodesDescription: "Selecteer statuscodes die als een succesvol antwoord worden beschouwd.",
+    passwordNotMatchMsg: "Het herhaalwachtwoord komt niet overeen.",
+    notificationDescription: "Wijs a.u.b. een melding toe aan de monitor(s) om het te laten werken.",
+    keywordDescription: "Zoek trefwoord in gewone html of JSON-response en het is hoofdlettergevoelig",
+    pauseDashboardHome: "Gepauzeerd",
+    deleteMonitorMsg: "Weet u zeker dat u deze monitor wilt verwijderen?",
+    deleteNotificationMsg: "Weet u zeker dat u deze melding voor alle monitoren wilt verwijderen?",
+    resoverserverDescription: "Cloudflare is de standaardserver, u kunt de resolver server op elk moment wijzigen.",
+    rrtypeDescription: "Selecteer het RR-type dat u wilt monitoren",
+    pauseMonitorMsg: "Weet je zeker dat je wilt pauzeren?",
+    Settings: "Instellingen",
+    Dashboard: "Dashboard",
+    "New Update": "Nieuwe update",
+    Language: "Taal",
+    Appearance: "Weergave",
+    Theme: "Thema",
+    General: "Algemeen",
+    Version: "Versie",
+    "Check Update On GitHub": "Controleer update op GitHub",
```

Comment:
Wouldn't "Controleer voor updates op GitHub" fit better?

Reply:
Thanks. Changed it. I was trying to keep the length of the string as close as possible to the English version. (Some sentences in Dutch are way longer than in English. Don't want to break the UI.

But this was minor, so changed 😊 thanks.